backend/src/controllers/course.js

```diff
@@ -18,13 +18,25 @@ exports.getCourse = function ({ params }, res) {
 
 /* Get all questions for a course */
 exports.getCourseQuestions = function ({ params, query }, res) {
-    responseHandler(questionModel.getQuestions(params.code, query.p), res)
+    // If query 'p' is not given just use p=1
+    let pageNumber = typeof query.p === 'undefined' ? 1 : parseInt(query.p)
+    // Check that pageNumber is an integer
+    if (isNaN(pageNumber)) {
+        throw Error('Page Number is not a number')
```

Comment:
What does this throw do currently? It should use the error handler to return some meaningful json to the frontend with error code 400.